tests/configfiles/DockerfileLintBearTest.py

```diff
@@ -23,7 +24,40 @@
 apt-get -y -qq upgrade
 """
 
+good_file_nodejs = """
+FROM ubuntu
+MAINTAINER coala
+LABEL Name coala-docker
+LABEL Version 0.1
+
+RUN apt-get -y -qq update
+RUN apt-get -y -qq upgrade
+RUN curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
+EXPOSE 5432
+CMD ["/usr/bin/node", "/var/www/app.js"]
+"""
+
+bad_file_nodejs = """
+FROM ubuntu
+
+apt-get -y -qq update
+apt-get -y -qq upgrade
+curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
+"""
+
+rule_file = os.path.join(os.path.dirname(__file__),
+                         'test_files',
+                         'default_rule.yaml')
 
 DockerfileLintBearTest = verify_local_bear(DockerfileLintBear,
                                            valid_files=(good_file,),
                                            invalid_files=(bad_file,))
+
+DockerfileLintBearRuleTest = verify_local_bear(DockerfileLintBear,
+                                               valid_files=(
+                                                   good_file_nodejs,),
+                                               invalid_files=(
+                                                   bad_file_nodejs,),
+                                               settings={'rule_file':
+                                                         os.path.abspath(
```

Comment:
Why do you need to use absolute path here?

Reply:
@anctartica Providing a relative path as a `str` doesn't work. And `os.path.join()` also returns an absoulte path. So, instead I used `abspath()` just to make it clear that an absoulte path is being provided here.

Reply:
okay, but if it is not required then you can remove it :)